### PR TITLE
fix bug for camera not pausing in scopeless mode

### PR DESCRIPTION
**Pull request by kkoetter**
Changed the toggling between `camera_loop` and `pause_loop` in `CameraProcess` to avoid camera getting stuck in camera loop for the mock camera. Issue with `--scopeless `was that the Camera was just entering `pause_loop` at initialisation of the GUI and afterwards was in `camera_loop` without a return. Code needs to be tested with hamamatsu camera to verify changes didn't crash anything. There are multiple `CameraMode` that are created but never checked such as `TRIGGERED`, `EXPERIMENT_RUNNING`. Could this potentially be simplified?

### CODE REVIEW

**Comment by kkoetter** [comment]
Update: Tested with hamamatsu in the light sheet computer. No apparent problems so far.

**Review by vigji — CHANGES_REQUESTED** [review]

File: sashimi/processes/camera.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
 class CameraMode(Enum):
     PREVIEW = 1
-    TRIGGERED = 2
-    EXPERIMENT_RUNNING = 3
+    #TRIGGERED = 2
```
[the remark this answers is not in the document]
Comment (vilim) [reply sashimi/processes/camera.py:24]: There should be no modes that are not used, and no commented lines

File: sashimi/processes/camera.py [hunk 2]
```diff
@@ -143,18 +143,18 @@ def run_camera(self):
...
-            if self.parameters.camera_mode == CameraMode.PAUSED:
-                self.pause_loop()
-            else:
-                self.camera.start_acquisition()
-                self.logger.log_message("Started acquisition")
-                self.camera_loop()
+            #if self.parameters.camera_mode == CameraMode.PAUSED:
+            self.pause_loop()
+            #else:
+            self.camera.start_acquisition()
```
[the remark this answers is not in the document]
Comment (vilim) [reply sashimi/processes/camera.py:149]: Also please remove the commented lines and write a small comment why the pause_loop is enough (basically inside of it the `if` you removed above is implemented.

**Review by vilim — APPROVED** [review]
Looks good, after these few changes it can be merged!

**Comment by kkoetter** [comment]
I now have removed the unnecessary `CameraMode`s and restructured the logger into the `camera_loop` and `pause_loop` to clean up the `run_camera` function. If thee is no more changes requested I would be happy for someone to review it again :)

**Review by vigji — APPROVED** [review]

**Pull request merged**